content/en/guides/hosting/iam/scim.md

```diff
@@ -746,32 +746,41 @@ POST /scim/Groups
 - **Method**: PATCH
 - **Description**: Update an existing team's membership list.
 - **Supported Operations**: `add` member, `remove` member
-- **Request Example**:
 
-Adding `dev-user2` to `wandb-devs`
+The remove operations follow RFC 7644 SCIM protocol specifications. Use the filter syntax `members[value eq "{user_id}"]` to remove a specific user, or `members` to remove all users from the team.
+
+**Request Example**:
+
+{{% alert color="info" %}}
+Replace `{team_id}` with the actual team ID and `{user_id}` with the actual user ID in your requests.
+{{% /alert %}}
+
+**Adding a user to a team**
```

Comment:
The formatting is fixed but now I am questioning these bold items. The first one isn't an example, it's just a note. I think the "Adding a user to a team" should be "Request example" with some para text like "This example adds a user to a team." Maybe line 752 is meant to be "Examples" plural. WDYT?